app/assets/stylesheets/nav.scss

```diff
@@ -46,14 +74,30 @@ $button-color: #1E90FF;
     }
   }
 }
+.job_deets{
+  font-family: Arial,sans-serif;
+  font-size: 10pt;
+  line-height: 1.3;
+}
+.sub-nav {
+  margin-left: 18%;
+}
+
+.panel{
+  background-color: #fff;
+  box-shadow: 0 1px 1px
+}
 
 .text_field {
   input {
-    height: 5000px;
-    width: 3000px;
   }
 }
-
+.tracking{
+  a {
+    color: purple;
```

Comment:
Color `purple` should be written in hexadecimal form as `#800080`<br>Color literals like `purple` should only be used in variable declarations; they should be referred to via variable everywhere else.